sdk/core/core-amqp/README.md

```diff
@@ -1,213 +1,119 @@
 # Azure Core AMQP client library for AMQP operations
 
-Library that provides common functionality for different **Azure** Javascript libraries using amqp protocol.
-Some of the common functionalities include:
-
-- CBS Auth
-- Request-Response link for sending request and receiving response over AMQP
-- Error translation of different AMQP error codes.
-- Creation of `ConnectionConfig` by parsing the connection-string
-- AMQP message header and message property transformation
-- AMQP message data transformation (encoding/decoding) that most AMQP based Azure services use for communication
-- RetryPolicy for retrying different operations
+Azure Core AMQP is a library that provides common functionality for **Azure** Javascript
+libraries that use [AMQP protocol](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-amqp-protocol-guide)
+like the ones for Azure Service Bus and Azure Event Hubs.
```

Comment:
Azure Service Bus and Azure Event Hubs could be links to those client libraries on GitHub (or, alternatively, on npmjs.com).